logging/cloud-client/src/main/java/com/example/logging/QuickstartSample.java

```diff
@@ -40,11 +47,12 @@ public static void main(String... args) throws Exception {
     String text = "Hello, world!";
 
     LogEntry entry = LogEntry.newBuilder(StringPayload.of(text))
+        .setSeverity(Severity.ERROR)
         .setLogName(logName)
         .setResource(MonitoredResource.newBuilder("global").build())
         .build();
 
-    // Writes the log entry
+    // Writes the log entry asynchronously
```

Comment:
Out of curiosity, do we do anything to improve the odds that logs get written even if the app crashes? Can we? Big try / catch in main?

Reply:
we do offer a logging.flush() but would n't help with the try / catch in main. 
I get your point though, so will look into it separately as an issue on `google-cloud-java`.